README.md

```diff
@@ -27,31 +27,8 @@ These plug-ins simplify integration of the Rollbar.NET Notifier into codebases t
 [![Reliability Rating](https://sonarcloud.io/api/project_badges/measure?project=rollbar-net&metric=reliability_rating)](https://sonarcloud.io/dashboard?id=rollbar-net)
 [![Maintainability Rating](https://sonarcloud.io/api/project_badges/measure?project=rollbar-net&metric=sqale_rating)](https://sonarcloud.io/dashboard?id=rollbar-net)
 [![Vulnerabilities](https://sonarcloud.io/api/project_badges/measure?project=rollbar-net&metric=vulnerabilities)](https://sonarcloud.io/dashboard?id=rollbar-net)
-[![Bugs](https://sonarcloud.io/api/project_badges/measure?project=rollbar-net&metric=bugs)](https://sonarcloud.io/dashboard?id=rollbar-net)
-[![Lines of Code](https://sonarcloud.io/api/project_badges/measure?project=rollbar-net&metric=ncloc)](https://sonarcloud.io/dashboard?id=rollbar-net)
-[![Duplicated Lines (%)](https://sonarcloud.io/api/project_badges/measure?project=rollbar-net&metric=duplicated_lines_density)](https://sonarcloud.io/dashboard?id=rollbar-net)
```

Comment:
Duplicated Lines of code badge is important code quality indicator. BTW, once I remove deprecated code this badge is expected to show 0% or fractionally close to it

Reply:
As explained above one can always see additional stats on SonarCloud page. I think keeping Security, Reliability, Maintainability, Vulnerabilities and Coverage is good enough to for readme.